test/test.dart

```diff
@@ -399,9 +399,18 @@ String generateRandomRealmPath() {
 }
 
 final random = Random();
+String _getRandom(int len, String chars) => List.generate(len, (index) => chars[random.nextInt(chars.length)]).join();
+
 String generateRandomString(int len) {
-  const chars = 'abcdefghjklmnopqrstuvwxuz';
-  return List.generate(len, (index) => chars[random.nextInt(chars.length)]).join();
+  return _getRandom(len, 'abcdefghjklmnopqrstuvwxuz');
```

Comment:
Not sure I understand the reasoning behind this change.

Reply:
I needed to generate strings containing unicode and not containing unicode symbols. That's why this method was refactored.